subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/AbstractInitIntegrationSpec.groovy

```diff
@@ -37,6 +37,7 @@ abstract class AbstractInitIntegrationSpec extends AbstractIntegrationSpec {
         file("settings.gradle") << """
             // This is here to prevent Gradle searching up to find the build's settings.gradle
         """
+        file("gradle.properties") << "\nkotlin.incremental.useClasspathSnapshot=false\n"
```

Comment:
A comment should link to the root cause issue at https://youtrack.jetbrains.com/issue/KT-57757/Reduce-classpath-snapshotter-memory-consumption

Reply:
This should be removed now that we do the same for all windows embedded tests

Reply:
Done, dropped the commit.